README.md

```diff
@@ -38,79 +38,85 @@ Create a `NotificationRequest` providing two documents: the act to notify and th
 
 
 ## How to run
+We provide a couple of different ways to start the `PnValidator`:
 
-### Prerequisites
-If you want to run the emulator starting from the source code, please follow these steps.
+- from the source code
+- as a [Docker](https://docker.com) image
 
-First, clone the repository using the preferred method (the next command uses SSH):
+### Run from the source code
 
-```shell
+This section describes how to start the `PnValidator` from the source code.
+
+Since this project runs with [Node.js](https://nodejs.org/en/) as specified in the [`.node_version`](.node-version) file, we strongly encourage [using nvm](https://github.com/nvm-sh/nvm).
+
+Here is what you need to do:
+
+1. clone the repository using the preferred method (the next command uses SSH):
+
+ ```shell
 git clone git@github.com:pagopa/pn-local-emulator.git
 ```
 
-Second, this project runs with [Node.js](https://nodejs.org/en/) and it has been developed with the version specified in the [`.node_version`](.node-version) file.
-
-We suggest [using nvm](https://github.com/nvm-sh/nvm) to install Node.js at the same version specified in the `.node_version` file.
+2. (Optional but strongly recommended) Install the Node.js runtime using `nvm` (please, make sure the path of `.node_version` is correct. The given commands assume you are in the repository folder.).
 
-```shell
+ ```shell
 # Install the version of Node.js specified in the .node_version file
 nvm install `cat .node-version`
 
-# Set the version of Node.js specified in the .node_version file
+ # Set the version of Node.js specified in the .node_version file
 nvm use `cat .node-version`
 ```
-Please, make sure the path of `.node_version` is correct. The given commands assume you are in the repository folder.
 
-### Run using Node.js
+3. Install the dependencies.
 
-Install the dependencies.
-
-```shell
+ ```shell
 npm install
 ```
 
-Generates code from the [OpenAPI](./openapi/index.yaml) specification.
+4. Generates code from the [OpenAPI](./openapi/index.yaml) specification.
 
-```shell
+ ```shell
 npm run generate
 ```
 
-Start the application.
+5. Start the `PnValidator`.
 
-```shell
+ ```shell
 npm run start
 ```
 
-### Run using Docker (Dockerfile)
+### Run as a Docker image
 
-The repository comes with a Dockerfile that you can use to run the application with Docker.
+Another option is to run the container image already available in the containers registry, as follows:
 
-Build the image.
+1. Pull the image from the container registry.
```

Comment:
The pull is optional, so we can remove it. The command `docker run -p 3000:3000 ghcr.io/pagopa/pn-local-emulator:latest` does the pull if needed

Reply:
Agreed. Simpler is better